openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -54,8 +54,9 @@ def list_entries(request, **kwargs):
                   types_of_assessment[type_of_assessment].lower())
 
     try:
-        curve_id = int(request.GET.get("curve",
-                                 TA.FRAGILITY))
```

Comment:
The second argument of ```get``` method is the default value to use when ```curve``` argument isn't set, the correct value is ```-1``` not ```TA.FRAGILITY```.